rosetta-source/src/main/rosetta/observable-asset-type.rosetta

```diff
@@ -129,6 +129,7 @@ type PriceSchedule extends MeasureSchedule: <"Specifies the price of a financial
     composite PriceComposite (0..1) <"(Optionally) Specifies the underlying price components if the price can be expressed as a composite: e.g. dirty price = clean price + accrued.">
     arithmeticOperator ArithmeticOperationEnum (0..1) <"(Optionally) When the price is to be understood as an operator to apply to an observable, i.e. a spread, multiplier or min/max.">
     cashPrice CashPrice (0..1) <"(Optionally when the price type is cash) Additional attributes that further define a cash price, e.g. what type of fee it is.">
+    priceScheduleReference PriceScheduleReferenceEnum (0..1) <"Defines a reference type to the price and how it can be applied.">
```

Comment:
I don't think this is the right design. We spent almost 2 years decoupling a universal  structural definition of price from the business context where a price can be used.  

In addition, could the qualification of the price as a purchase price or repurchase price be inferred from the properties of the trade event and its lineage. Given the fundamental role of price, please expand on what purchase and purchase price means and when they are used. I wondered if PriceTypeEnum could be the descriptor we need provided the new values are mutually exclusive from the exiting set.

Reply:
PurchasePrice and RepurchasePrice and not new types of prices in the context of how priceType is defined. It is not possible to infer the purchasePrice with a reference identifier.